src/libaktualizr/uptane/fetcher.cc

```diff
@@ -139,9 +133,9 @@ bool Fetcher::fetchVerifyTarget(const Target& target) {
 #ifdef BUILD_OSTREE
       KeyManager keys(storage, config.keymanagerConfig());
       keys.loadKeys();
-      std::function<void()> pause = std::bind(&Fetcher::checkPause, this);
-      data::InstallOutcome outcome =
-          OstreeManager::pull(config.pacman.sysroot, config.pacman.ostree_server, keys, target, pause, events_channel);
+      std::function<void()> check_pause = std::bind(&Fetcher::checkPause, this);
+      data::InstallOutcome outcome = OstreeManager::pull(config.pacman.sysroot, config.pacman.ostree_server, keys,
```

Comment:
Was it compiling here after you removed `pause`? Didn't know that it's ok to pass `int (*)(void)` instead of `std::function<void()>`.

Reply:
Yes that's here, I didn't know either...

Would have liked to find the relevant rule in the standard but I failed to do so.